### ISSUE TITLE
VL53L0X_multi_example.py API Status: -20 Control Interface Error

### ISSUE THREAD

**Issue by Shawn2448** [johnbryanmoore/VL53L0X_rasp_python#20]
If I use the single device example file I get flawless measurements which has proved my lasers to be functional.

However, I am trying to eventually utilize three of them and I only have one bus. Right now I am trying to use two lasers and I am trying to use the VL53L0X_multi_example.py file but I keep getting API Status: -20 Control Interface Error for both. Has anyone found a solution for this issue?

**Comment by TinManAkshay** [comment]
Yeah even I am getting the same error. I am using 5 of them, set on different i2c addresses. Other than that, one of sensor is found to be undetectable when I run the command sudo i2cdetect -y 1 and see what i2c addresses are connected to the Pi. 

There is some online source suggesting unplugging the sensor and then plugging it again to the power supply, resolves the issue. But it does not. (FYI: its not about rebooting the Pi). 

If anyone has an idea to resolve the matter, please do share your thoughts. 

Thanks,
Akshay

**Comment by kgdthomas** [comment]
Has there been any fixes to this?

**Comment by shinaaa2** [comment]
I want know it too...

**Comment by kokou-sekpona** [comment]
I got the same error with 2 sensors which worked fine. Please help me
